runtime-desktop/qtkeychain/autobuild/build

```diff
@@ -0,0 +1,36 @@
+abinfo "Building Qt5 version ..."
+abinfo "Creating build directory for the Qt5 build ..."
+mkdir -v build-qt5
+abinfo "Running CMake configure for the Qt5 build ..."
+cd build-qt5
+cmake "$SRCDIR" \
```

Comment:
Best to use CMake macros, as defined here:

https://github.com/AOSC-Dev/autobuild4/blob/master/templates/11-cmakeninja.sh